cylc/flow/xtrigger_mgr.py

```diff
@@ -279,12 +280,22 @@ def check_xtrigger(
                 fname,
                 f"'{fname}' not found in xtrigger module '{fname}'",
             )
+
         if not callable(func):
             raise XtriggerConfigError(
                 label,
                 fname,
                 f"'{fname}' not callable in xtrigger module '{fname}'",
             )
+        if func is not wall_clock:
+            # Validate args and kwargs against the function signature
+            # (but not for wall_clock because it's a special case).
```

Comment:
Can you add why wall_clock is a special case?

Reply:
I don't know why, but the `wall_clock` xtrigger has a different signature to the actual function.  Would that be sufficient explanation?